tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
     "compilerOptions": {
         "module": "commonjs",
+        "resolveJsonModule": true,
```

Comment:
**Callout:** Added to allow importing json files used in sourcecode. See [extensions.ts#L24-L25](https://github.com/IBM-Cloud/ibm-developer-extension-vscode/blob/feat/add-new-dev-cli-cmds/src/extension.ts#L24-L25) for usecase